app/assets/src/components/views/discovery/mapping/MapPreviewSidebar.jsx

```diff
@@ -218,10 +218,20 @@ export default class MapPreviewSidebar extends React.Component {
     );
   };
 
+  renderNoData = () => {
+    return (
+      <div className={cs.noData}>
+        Select a location to see summary info and samples.
```

Comment:
When does this happen (samples.length = 0)? 
Shouldn't by default just see all samples?

Reply:
oh yeah I broke that off as a separate task (pulling in what's already in DiscoveryView) and I think I'll add that soon in a separate PR